modules/jsoneditor.yml

```
@@ -0,0 +1,16 @@
+name: jsoneditor
+description: UI tool for managing JSON data
+repo: bestkolobok/nuxt-jsoneditor
+npm: nuxt-jsoneditor
+icon: 'jsoneditor.png'
+github: https://github.com/bestkolobok/nuxt-jsoneditor
+website: https://github.com/bestkolobok/nuxt-jsoneditor
+learn_more: https://github.com/bestkolobok/nuxt-jsoneditor
```

Learn more goes to the _integration_ site

```suggestion
learn_more: https://github.com/bestkolobok/vue3-jsoneditor
```